### ISSUE TITLE
Use pm.Censored in accelerated failure model

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pymc-examples#747]
**Notebook title**:
**Notebook url**:

Instead of using a Potential use Censored

**Comment by ricardoV94** [comment]
> Hey @ricardoV94 would it be alright if I worked on this?

Yes, you can claim any open issue by opening a PR. Until then it's up for grabs

**Comment by Dekermanjian** [comment]
Understood! Thank you, @ricardoV94 !
